docs/introduction.rst

```diff
@@ -82,27 +82,66 @@ approaches to this:
    output for the related images, we can deduce certain aspects of the
    model's behavior.
 
+
+.. note:: These descriptions apply only to *using the model after it has been created*; nothing is implied about how the model is constructed.
+
+.. note:: This distinction applies to both the *concept* and *implementation*. A model could support white box reasoning but lack the APIs to access the internal state; conversely, it might provide complete API access yet provide no guidance on how that state influences the output. **API access to the model does not automatically imply white box XAI.**
```

Comment:
Minor and maybe subjective: the use of "XAI" and the end here didn't make sense / flow for me. Maybe instead the following?
```suggestion
.. note:: This distinction applies to both the *concept* and *implementation*. A model could support white box reasoning but lack the APIs to access the internal state; conversely, it might provide complete API access yet provide no guidance on how that state influences the output. **API access to the model does not automatically imply white box explainability.**
```

Reply:
I think what we are trying to capture is the difference between "white box" and "black box" testing (but essentially from a ML versus software perspective)? e.g. see comparison tables [here](https://www.geeksforgeeks.org/differences-between-black-box-testing-vs-white-box-testing/) and [here](https://www.guru99.com/back-box-vs-white-box-testing.html)